scipy/interpolate/_rgi.py

```diff
@@ -53,9 +53,9 @@ class RegularGridInterpolator:
 
     method : str, optional
         The method of interpolation to perform. Supported are "linear",
-        "nearest", "slinear", "cubic", and "quintic". This parameter will
-        become the default for the object's ``__call__`` method.
-        Default is "linear".
+        "nearest", "slinear", "cubic", "quintic" and "pchip". This
+        parameter will become the default for the object's ``__call__``
+         method. Default is "linear".
```

Comment:
Refguide check is failed because of this unnecessary space: https://dev.azure.com/scipy-org/SciPy/_build/results?buildId=18747&view=logs&j=ede89041-5e8e-5201-ce5c-f68c2cbf02dd&t=bf6d2fae-c4b4-5aaf-e29f-d3092465da33&l=109
```suggestion
        method. Default is "linear".
```